_examples/hi/test.py

```diff
@@ -167,8 +167,8 @@
 
 ## test Couple.__init__
 print("--- Couple.__init__")
-c = hi.Couple(hi.Person("p1", 42))
-print(c)
+#c = hi.Couple(hi.Person("p1", 42))
```

Comment:
why doesn't this work anymore?
(is there a FIXME somewhere?)

Reply:
pybindgen doesn't automatically support varargs.. so it needs the full 2 args per the Go signature.  more generally, retaining the Go signature seems like a good idea, right?